packages/dds/tree/src/shared-tree-core/editManager.ts

```diff
@@ -133,23 +140,40 @@ export class EditManager<
 		public readonly changeFamily: TChangeFamily,
 		public readonly localSessionId: SessionId,
 		private readonly mintRevisionTag: () => RevisionTag,
+		private readonly logger?: ITelemetryLoggerExt,
 	) {
 		this.trunkBase = {
 			revision: "root",
 			change: changeFamily.rebaser.compose([]),
 		};
 		this.sequenceMap.set(minimumPossibleSequenceId, this.trunkBase);
+
+		if (this.logger !== undefined) {
+			const THRESHOLD = 1000;
```

Comment:
If we need to keep a constant in this file, let's make it a module-scope one instead of redeclaring a variable here.